src/frontend/src/types/types.ts

```diff
@@ -29,6 +36,12 @@ export interface IGroup {
     project: string;
 }
 
+export interface IPotentialInvite {
```

Comment:
we should really touch up the naming around the invite/application but I don't have the time right now. I think invite makes more sense than application, as we invite students to group (not the other way around)